### PR TITLE
Update mgmt SDK to track2

### PR DESCRIPTION
**Pull request by v-jiaodi**
Update Resource and App Service mgmt SDK from track1 to track2.

@qiaozha for notification.

### CODE REVIEW

File: cleanup.js [hunk 1]
```diff
@@ -49,17 +60,14 @@ function _validateParameters() {
 }
 
 //Entrypoint of the cleanup script
-msRestAzure.loginWithServicePrincipalSecret(clientId, secret, domain, function (err, credentials) {
-  if (err) return console.log(err);
+async function main(){
+  const credentials = new DefaultAzureCredential();
```
Comment (qiaozha) [inline cleanup.js:64]: use ClientSecretCredential in this case

File: cleanup.js [hunk 2]
```diff
@@ -49,17 +60,14 @@ function _validateParameters() {
...
-    console.log('\nDeleting the resource group can take few minutes, so please be patient :).');
-    deleteResourceGroup(function (err, result) {
-      if (err) return console.log('Error occured in deleting the resource group: ' + resourceGroupName + '\n' + util.inspect(err, { depth: null }));
-      console.log('Successfully deleted the resourcegroup: ' + resourceGroupName);
-    });
-  });
-});
\ No newline at end of file
+  websiteClient = new WebSiteManagementClient(credentials, subscriptionId);
+  await deleteWebSite();
```
[the remark this answers is not in the document]
Comment (v-jiaodi) [reply cleanup.js:67]: I think it is reasonable to optimize the code in this way. Of course, this is only my suggestion.

[the remark this answers is not in the document]
Comment (qiaozha) [reply cleanup.js:67]: As there's other callback related change in this PR, let's defer the approval to the service team if possible.

[the remark this answers is not in the document]
Comment (v-jiaodi) [reply cleanup.js]: I have updated it.

File: cleanup.js [hunk 3]
```diff
@@ -49,17 +49,22 @@ function _validateParameters() {
...
-    if (err) return console.log('Error occured in deleting the website: ' + websiteName + '\n' + util.inspect(err, { depth: null }));
+  websiteClient = new WebSiteManagementClient(credentials, subscriptionId);
+  deleteWebSite().then((result) => {
     console.log('Successfully deleted the website: ' + websiteName);
     console.log('\nDeleting the resource group can take few minutes, so please be patient :).');
-    deleteResourceGroup(function (err, result) {
-      if (err) return console.log('Error occured in deleting the resource group: ' + resourceGroupName + '\n' + util.inspect(err, { depth: null }));
+    deleteResourceGroup().then((result) => {
       console.log('Successfully deleted the resourcegroup: ' + resourceGroupName);
+    },(err)=>{
```
Comment (qiaozha) [inline cleanup.js:61]: .catch(err => ) ?

Comment (qiaozha) [inline cleanup.js:61]: .catch(err => ) ?

File: cleanup.js [hunk 4]
```diff
@@ -49,17 +49,22 @@ function _validateParameters() {
...
+      console.log('Error occured in deleting the resource group: ' + resourceGroupName + '\n' + util.inspect(err, { depth: null }));
     });
+  },(err)=>{
```
Comment (qiaozha) [inline cleanup.js:64]: same here.

Comment (qiaozha) [inline cleanup.js:64]: same here.

File: cleanup.js (hunk 3 above)
Comment (v-jiaodi) [reply cleanup.js:61]: It's like as follow:
```js
promise.then(value => {
  // fulfillment
}, reason => {
  // rejection
});
```

**Review by qiaozha — APPROVED** [review]
LGTM

**Comment by jongio** [comment]
Please resolve all conversations.

**Comment by v-xuto** [comment]
@jongio We have resolved all conservations.

**Pull request merged**